### ISSUE TITLE
[TW-1872] Project names beginning with "startR" fail to evaluate

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1885]
_Leon Feng on 2016-12-02T16:37:07Z says:_ 

 This is a head scratcher. It seems any project name in the form of `^startR.*$` causes `task add` to fail (see screenshot). Replace "start" with "tart" or "art" and it works fine. Weird, huh?

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-12-02T16:37:07Z
Modified: 2017-01-24T17:08:12Z
```

**Comment by taskwarrior** [comment]
_Leon Feng on 2016-12-02T16:49:20Z says:_ 

 Oh and `task mod` has the same issue. Can't test the other commands since I can't create a task with a problematic project name.

**Comment by taskwarrior** [comment]
_Peter Rochen on 2016-12-10T08:34:51Z says:_ 

 Or begins with "id". project:idfoobar.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-12-10T16:30:35Z says:_ 

 This is a bug where the command line parser is not checking the end boundaries properly.  Good catch, thanks guys.

**Comment by pbeckingham** [comment]
![tw-1872_screen shot 2016-12-03 at 00 32 59](https://user-images.githubusercontent.com/7963/36643716-a232460a-1a1d-11e8-8322-299469986737.png)

**Comment by jeffballard** [comment]
For what its worth "end" seems to have the same problem.  (endtimes, endy, and so on can't be project names)

**Comment by tbabej** [comment]
Fixed in 2.5.2, 2.6.0.
